config/default/manager_patch.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        envFrom:
+        - secretRef:
+            name: redsky-manager
+            optional: true
```

Comment:
I would rather not have this moved into `config/default` as it that would include it into other downstream kustomizations.

Reply:
Should we try to inject this conditionally if there is a complete redskyctl config file?

Reply:
Sure: this is here so we could have one static manifest for both the OSS and Enterprise editions, but having it optional has caused some unique timing related issues that are hard to debug.